Demo/Application/Features/UnionPay/BraintreeDemoUnionPayViewController.m

```diff
@@ -111,7 +111,11 @@ - (void)viewDidLoad {
 - (void)enroll:(__unused UIButton *)button {
     self.progressBlock(@"Enrolling card");
 
-    BTCard *card = [[BTCard alloc] initWithNumber:self.cardForm.number expirationMonth:self.cardForm.expirationMonth expirationYear:self.cardForm.expirationYear cvv:self.cardForm.cvv];
+    BTCard *card = [BTCard new];
+    card.number = self.cardForm.number;
+    card.expirationMonth = self.cardForm.expirationMonth;
+    card.expirationYear = self.cardForm.expirationYear;
+    card.cvv = self.cardForm.cvv;
 //    card.shouldValidate = YES;
```

Comment:
Should this be included or removed?

Reply:
We left it in - since uncommenting this line is an easy way to toggle on/off vaulting for this demo feature. We can delete though if it's more clear.

Reply:
Ah ok that makes sense, no need to remove it if it makes the demo app easier to use.

Reply:
Looking at this again, I actually think we should just remove it. It's not hard for someone to just add that line back in if they need it to test something.